src/runtime/composables/useFormGroup.ts

```diff
@@ -38,9 +46,12 @@ export const useFormGroup = (inputAttrs?: InputAttrs) => {
     }, 300)
 
     return {
+      inputId: formGroup?.inputId,
+      name: computed(() => inputProps?.name ?? formGroup?.name),
+      size: computed(() => inputProps?.size ?? formGroup?.size?.value ?? appConfig.ui.input.default.size),
```

Comment:
Unfortunately `appConfig.ui.input.default.size` won't work here as the default config no longer lives in the `appConfig`, it's only the one from the user. You can take a look at the `FormGroup`: https://github.com/nuxt/ui/pull/704/files#diff-82ffaa8fe9c377bdfd6259594303b0111befc57be8db4b95e1e1d4cd0d3eb56dR30

`import { formGroup } from '#ui/ui.config'`

Reply:
Good catch! I looked into it, looks like it's going to be a bit more complicated than I imagined, the default value needs to be set to the input's default (not the form group), it might make more sense move this back into each components

Reply:
Or you can take the default from the `input` as most of the other form elements inherit from its config. This could lead to an edge-case if the user changes the config entirely but I think its good enough.

Reply:
I fixed it by providing the configuration object in the composable to avoid edge-cases, what do you think?